src/core/thread/fiber.d

```diff
@@ -32,6 +32,8 @@ version (LDC)
         import ldc.sanitizers_optionally_linked;
     }
 }
+else
+    enum assumeUsed;
```

Comment:
Should IMO be private and not a type, e.g. `private enum assumeUsed = null`.

Reply:
Fixed.